{angle_time: [0.0, 0.25, 0.5, 1.0, 1.75, 3.5, 5.25, 7.0], 
angle: [0.0, 0.165, 0.36, 0.81, 1.56, 3.17, 4.8, 6.28], 
distance: [0.0, 0.022, 0.05, 0.11, 0.25, 0.50],
distance_time: [0.0, 0.25, 0.5, 1.0, 2.0, 4.0],
area_limit: [0.43, 1.24, 0.22, -1.02]}
